resiliency/service-to-service-resiliency.md

```diff
@@ -0,0 +1,159 @@
+```
+
+##### Checkout Service: 
+```bash
+cd ../service_invocation/csharp/http/checkout
+dotnet restore
+dotnet build
+dapr run  --app-id checkout --config ../config.yaml --components-path ../../../components/ --app-protocol http --dapr-http-port 3500 -- dotnet run
+```
+
+### Go example:
+##### Order Processor Service: 
+```bash
+cd ../service_invocation/go/http/order-processor
+go build .
+dapr run --app-port 6001 --app-id order-processor --app-protocol http --dapr-http-port 3501 -- go run .
+```
+
+##### Checkout Service: 
+```bash
+cd ../service_invocation/go/http/checkout
+go build .
+dapr run  --app-id checkout --config ../config.yaml --components-path ../../../components/  --app-protocol http --dapr-http-port 3500 -- go run .
+```
+
+### Java example:
+##### Order Processor Service: 
+```bash
+cd ../service_invocation/java/http/order-processor
+mvn clean install
+dapr run --app-id order-processor --app-port 9001 --app-protocol http --dapr-http-port 3501 -- java -jar target/OrderProcessingService-0.0.1-SNAPSHOT.jar
+```
+
+##### Checkout Service: 
+```bash
+cd ../service_invocation/java/http/checkout
+mvn clean install
+dapr run --app-id checkout --config ../config.yaml --components-path ../../../components/ --app-protocol http --dapr-http-port 3500 -- java -jar target/CheckoutService-0.0.1-SNAPSHOT.jar
+```
+
+### JavaScript example:
+##### Order Processor Service: 
+```bash
+cd ../service_invocation/javascript/http/order-processor
+npm install
+dapr run --app-port 5001 --app-id order-processor --app-protocol http --dapr-http-port 3501 -- npm start
+```
+
+##### Checkout Service: 
+```bash
+cd ../service_invocation/javascript/http/checkout
+npm install
+dapr run  --app-id checkout --config ../config.yaml --components-path ../../../components/ --app-protocol http --dapr-http-port 3500 -- npm start
+```
+
+### Python example:
+##### Order Processor Service: 
+```bash
+cd ../service_invocation/python/http/order-processor
+pip3 install -r requirements.txt 
+dapr run --app-port 8001 --app-id order-processor --app-protocol http --dapr-http-port 3501 -- python3 app.py
+```
+
+##### Checkout Service: 
+```bash
+cd ../service_invocation/python/http/checkout
+pip3 install -r requirements.txt 
+dapr run  --app-id checkout --config ../config.yaml --components-path ../../../components/ --app-protocol http --dapr-http-port 3500 -- python3 app.py
+```
+
+### Expected output: 
+Once both services are running, the `order-processor` service will recieve orders from the `checkout`service continuously using Dapr's service invoke API.
+
+##### `order-processor` output:
+```bash
+== APP == Order received: { orderId: 1 }
+== APP == Order received: { orderId: 2 }
+== APP == Order received: { orderId: 3 }
+```
+
+### Simulate ann application failure by stopping the `order-processor` service  
+
+Simulate a system failure by stopping the `order-processor` service running in your second terminal shell.
```

Comment:
Could we be more explicit "Run <this> in a new terminal window"?  Let's pick Terminal or Shell term and stick with it.  I believe we use terminal window in others.

Reply:
done